linesUtil.js

```diff
@@ -0,0 +1,52 @@
+var defaultChar = '-';
+var defaultLength = 30;
+
+function getChar(char, errLogFunc) {
+	if (!char) {
+		char = defaultChar;
+	} else if (typeof(char) == 'object') {
+		errLogFunc("ERROR: the line() functions can't accept objects!");
+		char = defaultChar; //continue working with default behaviour.

```

Comment:
this is the only behavioural change I made. Instead of not logging the line and only logging an error, I log the error, but continue logging the line with the default character.